### ISSUE TITLE
Feature request: Ironing bottom layer option

### ISSUE THREAD

**Issue by EW00D** [SoftFever/OrcaSlicer#4183]
### Is there an existing issue for this feature request?

- [X] I have searched the existing issues

### Is your feature request related to a problem?

I`m trying to get the bottom of my print to not show the print lines. But whit the current options available I can’t get the entry bottom smooth.

![IMG_20240215_094234](https://github.com/SoftFever/OrcaSlicer/assets/31928688/75c40903-846b-4cb5-911e-d84c88fe3bf0)

![IMG_20240215_094210](https://github.com/SoftFever/OrcaSlicer/assets/31928688/d8de6e20-72c6-45cb-bbdf-0ebc8fc9a6f2)

### Which printers will be beneficial to this feature?

All

### Describe the solution you'd like

It would be nice to have an extra option in "Ironing type" to Bottom/Top layer.
So that the first layer does not show any print lines.

### Describe alternatives you've considered

I can almost have my print show no print lines at the bottom by changing the following settings:

- Layer height: 0.08mm
- First layer height: 0.08mm
- First layer width: 0.2mm
- Bottom surface flow rate: 2
- Only one wall on first layer
- First layer speed: 5mm/s
- First layer infill: 5mm/s
- Initial layer travel speed: 50%

But the start corner always show a few print lines.
Probably bad adhesion to the bed because of the "Layer height" and "First layer width".

Basically I want to have the first layer on 0.2mm and the First layer width on 0.1mm. But that is not possible.
